app/core/backup.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import fs from 'fs'
+import { getStorage } from './storage'
+
+export async function backupAccounts(
+  path: string,
+  chain: string,
+  callbackSuccess?: () => void,
+  callbackError?: (errorMsg: string) => void,
+) {
+  try {
+    const keyStorage = chain === 'neo2' ? 'userWallet' : 'n3UserWallet'
+    const accounts = await getStorage(keyStorage)
+    fs.writeFile(path, JSON.stringify(accounts), errorWriting => {
+      if (errorWriting && callbackError) {
+        callbackError(errorWriting.message)
+      } else if (callbackSuccess) {
+        callbackSuccess()
+      }
+    })
+  } catch (error) {
+    throw error
+  }
```

Comment:
It's redundant, you're catching an error to throw it, it's the same thing not catching it. So I recommend removing the try catch block